composer.json

```diff
@@ -11,38 +12,38 @@
     ],
     "require": {
         "php": ">=7.2",
-        "webonyx/graphql-php": "^0.13.4",
-        "psr/container": "^1",
-        "doctrine/annotations": "^1.7.0",
+        "ext-json": "*",
+        "doctrine/annotations": "^1.10.4",
         "doctrine/cache": "^1.8",
-        "thecodingmachine/class-explorer": "^1.1.0",
-        "psr/simple-cache": "^1",
+        "composer/package-versions-deprecated": "^1.8",
         "phpdocumentor/reflection-docblock": "^4.3 || ^5.0",
-        "phpdocumentor/type-resolver": "^1.0.1",
-        "psr/http-message": "^1",
-        "webmozart/assert": "^1.4",
-        "symfony/cache": "^4.3 | ^5",
-        "thecodingmachine/cache-utils": "^1",
-        "ocramius/package-versions": "^1.4 || ^2",
```

Comment:
Was this no longer needed?

Reply:
I replaced it with its fork to get rid of errors. I already did this in graphqlite-bundle